Packs/PagerDuty/Integrations/PagerDuty/PagerDuty.py

```diff
@@ -116,7 +113,7 @@ def http_request(method, url, params_dict=None, data=None, json_data=None, addit
         raise
 
 
-def translate_severity(sev):
+def translate_severity(sev):  # pragma: no cover
```

Comment:
Need a unit test

Reply:
The fact that `'high'` is all lower case, and `'Low'` is camel case raises a brow.
I'd check against a `sev.lower()`